### ISSUE TITLE
DP #59: Blogchain

### ISSUE THREAD

**Issue by xicubed** [district0x/district-proposals#59]
**Name**: Blogchain

**Purpose**: Open, decentralized blogging platform. Record human history.

**Description**: Too many times blogs come and go, war may destory a city. Medium's (and others) walls are closing in. Blogging software changes, comes and goes. Shouldn't the worlds blogs be recorded, syndicated, searchable, taggable, commentable and distributed for all? Hard for me to envision the scope but the typical features you'd see on Medium or WordPress.com.

**Comment by AngeloAdam** [comment]
please send me a direct message on district slack regarding your proposal. @angelo

**Comment by xicubed** [comment]
I'm @mal on slack.

**Comment by AngeloAdam** [comment]
Sent you a message on slack.

**Comment by mckmuze** [comment]
I'm going to close this for now. If you would like to elaborate on this proposal or add mock ups we can reopen your proposal.

**Comment by xicubed** [comment]
I would think it is fairly intuitive. The text for blogs could be stored in the blockchain. Very much like this "twitter" clone you could have a "medium" clone which held blog posts. https://medium.com/@matus.lestan/how-to-create-decentralised-apps-with-clojurescript-re-frame-and-ethereum-81de24d72ff5 I'm not too concerned with the UI or the functionality but that the blogs get preserved. Could start very simply and add features over time.

**Comment by Bradymck** [comment]
Nice, I will open your proposal back up. I agree this would be an awesome app to use.

**Comment by xicubed** [comment]
Thanks. I envision markdown as the payload/editor but could be anything really.

**Comment by WarKitteh** [comment]
I've been looking to get involved with dis0x but my ethereum dapp developing education is limited. My idea for a district was to have one very similar to yours but the viewers of the blog content would mine while they are viewing it. Let me know what you think and how I could get involved.

**Comment by shyblugs** [comment]
@WarKitteh how would a blog viewer mine by reading a blog?

**Comment by WarKitteh** [comment]
The same way someone runs CoinHive on their blog. The reader is reading or watching the content and their computer is mining it while it views the content.

**Comment by shyblugs** [comment]
Can you send a link to coinHive

**Comment by WarKitteh** [comment]
absolutely. Again this is a project I would like to contribute to so I would love to help make this a real thing, what ever I can do.
coinhive: https://coinhive.com/

**Comment by WarKitteh** [comment]
I guess the district would be more of an amateur content district where there is no middle man that owns your piece of work ex; article, video, or presentation. you own the work, and you get paid in mining by the people viewing the content and there is no one to mess with that.

**Comment by xicubed** [comment]
Thanks for the comments, interesting ideas. I mean payments could be maybe an extension or something but It should be deployable either with or without it. At it's core, blogging should be free, for producers or consumers.
